### PR TITLE
add VMware Photon OS docs

### PR DESCRIPTION
**Pull request by frapposelli**
this goes hand in hand with docker-library/official-images#1332

### CODE REVIEW

File: photon/content.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# VMware Photon OS
+
+%%LOGO%%
+
+Photon OS is a technology preview of a minimal Linux container host. It is designed to have a small footprint and boot extremely quickly on VMware platforms. Photon OS is intended to invite collaboration around running containerized applications in a virtualized environment.
+
+Photon contains `tdnf`, a new, open-source, yum-compatible package manager that will help make the system as small as possible, but preserve the robust yum package management capabilities.
```
Comment (tianon) [inline photon/content.md]: Guys around the office said you should've named it "cannon" or "torpedo" instead -- total missed opportunity here. :smile:

File: photon/license.md [hunk 2]
```diff
@@ -0,0 +1 @@
+View [license information](https://github.com/vmware/photon/blob/master/LICENSE) for the software contained in this image.
```
Comment (tianon) [inline photon/license.md]: Bahahahaha `Sorry about that, but we can’t show files that are this big right now.`, that's epic.  :+1:

File: photon/content.md (hunk 1 above)
Comment (frapposelli) [inline photon/content.md]: photon cannon sounds pretty badass, totally going to use it :D

**Comment by tianon** [comment]
LGTM

**Comment by frapposelli** [comment]
:tada:

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**